test/authority_test.go

```diff
@@ -80,15 +80,16 @@ func runUnixTest(t *testing.T, address, target, expectedAuthority string, dialer
 	}
 }
 
-// TestUnix does end to end tests with the various supported unix target
-// formats, ensuring that the authority is set to localhost in every case.
-func (s) TestUnix(t *testing.T) {
-	tests := []struct {
-		name      string
-		address   string
-		target    string
-		authority string
-	}{
+type authorityTest struct {
+	name           string
+	address        string
+	target         string
+	authority      string
+	expectedTarget string
```

Comment:
Go uses "want".  How about `dialTargetWant`?

Reply:
Changed it to `dialTargetWant`